### PR TITLE
Fixed malfunction in user management

### LINKED ISSUE #3150: User Management Malfunction
image

Even though the user is online, it is saying last seen,

### PR DESCRIPTION
**Pull request by cp-Coder**
# Updates

- [x] Fixes #3150
- [x] Removed `Last Online Status` for `Logged in` user

### CODE REVIEW

File: src/Components/Users/ManageUsers.tsx [hunk 1]
```diff
@@ -325,23 +325,32 @@ export default function ManageUsers() {
...
-                    ></span>
-                    <span className="pl-2">
-                      {user.last_login
-                        ? moment(user.last_login).fromNow()
-                        : "Never"}
-                    </span>
+                    {user.last_login &&
+                    moment()
+                      .subtract(5, "minutes")
+                      .isBefore(user.last_login) ? (
```
Comment (developedBySJ) [inline src/Components/Users/ManageUsers.tsx]: Remove duplicate code.
Use variables.

File: src/Components/Users/ManageUsers.tsx [hunk 2]
```diff
@@ -325,23 +325,32 @@ export default function ManageUsers() {
...
+                    ) : (
+                      <>
+                        <span>Last Online: </span>
+                        <span
+                          aria-label="Online"
+                          className={
+                            "shrink-0 inline-block h-2 w-2 rounded-full " +
+                            (moment()
+                              .subtract(5, "minutes")
+                              .isBefore(user.last_login)
```
Comment (developedBySJ) [inline src/Components/Users/ManageUsers.tsx]: Remove duplicate code.
Use variables.

**Review by developedBySJ — CHANGES_REQUESTED** [review]

**Review by developedBySJ — APPROVED** [review]
LGTM

**Comment by nihal467** [comment]
@cp-Coder 
image

Even though the user is logged out 5 minutes ago, it is saying online

**Comment by aeswibon** [comment]
@nihal467 I have tested this on another user. I didn't encounter the problem mentioned above.

File: src/Components/Users/ManageUsers.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { useCallback, useState } from "react";
+import React, { useCallback, useState } from "react";
```
Comment (Ashesh3) [inline src/Components/Users/ManageUsers.tsx:1]: Importing `React` is not required here, we're already importing the necessary functions.

```suggestion
import { useCallback, useState } from "react";
```

Tested and working fine.

**Review by Ashesh3 — APPROVED** [review]
Minor optional change, otherwise LGTM!

**Comment by nihal467** [comment]
test approved

File: src/Components/Users/ManageUsers.tsx (hunk 3 above)
Comment (aeswibon) [reply src/Components/Users/ManageUsers.tsx:1]: I have added `React` to fix Deepscan issue

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**